src/AssemblerConflictReadGraph.cpp

```diff
@@ -46,7 +46,7 @@ void Assembler::createConflictReadGraph(
 
     // Initialize the conflict read graph.
     conflictReadGraph.createNew(largeDataName("ConflictReadGraph"), largeDataPageSize);
-    conflictReadGraph.createVertices(readCount());
+    conflictReadGraph.createVertices(reads.readCount());
```

Comment:
Perhaps it would have made sense to keep `Assembler::readCount()` and let it simply forward to `class Reads`. Then code like this would not have been affected.

Reply:
But I am not proposing that we make this change now.